py/examples/tour.py

```diff
@@ -263,11 +289,16 @@ def make_blurb(q: Q):
     # HACK: Recreate dropdown every time (by dynamic name) to control value (needed for next / prev btn functionality).
     items = [ui.dropdown(name=q.args['#'] or default_example_name, width='300px', value=example.name, trigger=True,
                          choices=[ui.choice(name=e.name, label=e.title) for e in catalog.values()])]
+    buttons = []
+    mobile_buttons = []
     if example.previous_example:
-        items.append(ui.button(name=f'#{example.previous_example.name}', label='Previous'))
+        buttons.append(ui.button(name=f'#{example.previous_example.name}', label='Previous'))
+        mobile_buttons.append(ui.button(name=f'#{example.previous_example.name}', label='Prev'))
     if example.next_example:
-        items.append(ui.button(name=f'#{example.next_example.name}', label='Next', primary=True))
-    blurb_card.items = items
+        buttons.append(ui.button(name=f'#{example.next_example.name}', label='Next', primary=True))
+        mobile_buttons.append(ui.button(name=f'#{example.next_example.name}', label='Next', primary=True))
+    blurb_card.items = items + buttons
+    q.page['mobile_blurb'].items = [ui.inline(direction='row', justify='center', items=items + mobile_buttons)]
```

Comment:
Let's simplify this. No need for extra `buttons` and `mobile_buttons` as it seems like there is a difference only in `label` (correct me if I am wrong).

GH won't let me add suggestions on deleted lines so here is the whole code.

```py
    if example.previous_example:
        items.append(ui.button(name=f'#{example.previous_example.name}', label='Prev'))
    if example.next_example:
        items.append(ui.button(name=f'#{example.next_example.name}', label='Next', primary=True))
    q.page['mobile_blurb'].items = [ui.inline(direction='row', justify='center', items=items)]
```

Reply:
I agree. Btw, is my robust implementation the only way how to achieve different labels for smaller screens in Wave?

Reply:
[Robustness](https://en.wikipedia.org/wiki/Robustness_(computer_science)) means something different. If the labels really need to be different, I would go for a label update where needed, e.g. `q.page['mobile_blurb'].items....label = 'Prev'`. This way all there is needed is a single line and no extra variables.